modulemd/include/modulemd-2.0/modulemd-module.h

```diff
@@ -304,4 +305,34 @@ modulemd_module_get_defaults (ModulemdModule *self);
 ModulemdTranslation *
 modulemd_module_get_translation (ModulemdModule *self, const gchar *stream);
 
+
+/**
+ * modulemd_module_get_obsoletes:
+ * @self: (in): This #ModulemdModule object.
+ *
+ * Returns: (transfer none) (element-type ModulemdObsoletes): All obsoletes attached to this module.
```

Comment:
If you are doing `(transfer none)`, please indicate that this is returning the internal data and must not be modified or freed.